### PR TITLE
Expedition items

### PR DESCRIPTION
**Pull request by halfacandan**
Add Expedition Items

### CODE REVIEW

**Comment by halfacandan** [comment]
Just thinking that we could set up a webhook from Git into the Discord to let people know when this repo is updated. Something like this: https://gist.github.com/SGTGunner/50d6a3cc0d489cf779f77695ba3e22ea

**Review by omriharel — APPROVED** [review]
Looks great! We can add the webhook too, that sounds like a good idea. Let's set it up and I'll merge right after to test it

**Pull request merged**
